draft-ietf-mls-protocol.md

```diff
@@ -1264,7 +1270,8 @@ root node of the ratchet tree.
 ## Remove
 
 A Remove message is sent by a group member to remove one or more
-members from the group.
+members from the group. A member MUST NOT use a Remove message to
+remove themselves from the group.
```

Comment:
The receiver POV here seems more useful, i.e., "If a member of a group receives a Remove message where the `removed` index is equal to the `sender` index, then the recipient MUST reject the message as malformed."  That's not to say we can't have both, though.

Might tee this up better if you said "remove one or more **other** members from the group" (emphasis only for here, not for the doc).